### DISCUSSION TITLE
reviewers:@me filter

### DISCUSSION THREAD

**Discussion by VojtaStanek** [community/community#16777] [Projects and Issues]
Hi!

`assignee:@me` works but it doesn't work for reviewers.

**Comment by kiml42** [comment]
This would be really useful. I've got a view in our github project for all my PRs using the `assignee:@me` filter. It would be great to be able to use the `reviewers:@me` filter as well to have a view of everything I need to keep track of. Instead I have a separate view using `reviewers:kiml42` but that only works for me, so if the rest of the team want that we'd end up with dozens of extra views that are all doing the same thing.

**Reply by mikaylagarvey** [reply]
Yes, we're looking for this, too! `assign:@me -status:"Done"` works for now. However... As we begin to move away from Monday.com, we're going to need a more robust way to see what specific Reviewers have to complete. This is an easy win for PMs moving to this tool.

**Comment by zaporylie** [comment]
In fact `reviewers:@me` works for me well, but:
- one can't combine `assignee:@me reviewers:@me` to show all issues/prs both assigned to you and requested review from you
- it won't work for all the reviews requested from one as member of the team. Ie. if review was requested from @backend team, which you are part of, you won't see it on your board
